### ISSUE TITLE
Response History's Calendar & Map don't update when tapping the refresh button

### ISSUE THREAD

**Issue by ctrance** [ohmage/android-lib#38]
I deleted a response from the server then tapped refresh button from Response History page (when I tap the button, the loading indicator doesn't show). The counter on the calendar and the map didn't update, it only updated when I go to the dashboard then go back to Response History.

**Comment by ctrance** [comment]
@cketcham - This issue remains unresolved after testing the version 2.15.3 (22702c8) downloaded from web.ohmage.org/ohmage/apk. Is that version old? If yes, can you please send me the latest apk?

**Comment by cketcham** [comment]
That version should work. It tested and it is working for me. What are the steps you are doing that cause it to not remove the response that you deleted?

**Comment by ctrance** [comment]
I was testing the steps below on the emulator (google API 4.2.2) 
1. Uploaded new response from the ohmage app
2. Logged in to front end then deleted the newly uploaded response.
3. Went back to ohmage app and clicked 'refresh' button from 'Response History'
4. The counter didn't refresh from 1 to 0.

Note: I even waited for 2 minutes, then clicked 'refresh' again but the counter didn't reset. When I clicked the counter only then 'No Response' shows.

**Comment by cketcham** [comment]
I'm wondering if it is an issue with the emulator? could you try on an actual device?

**Comment by ctrance** [comment]
I tried it on the actual phone and after clicking refresh the counter still didn't refresh but when I click the counter it says 'no response'. When I switch to map then go back to calendar only then the counter resets.

**Comment by cketcham** [comment]
maybe try uninstalling and reinstalling the latest apk from web.ohmage.org/ohmage/apk. I can't seem to reproduce the same problem you are experiencing. Does the progress spinner show next to the refresh button?

**Comment by ctrance** [comment]
@cketcham  I reinstalled and the calendar is updating now after I clicked 'refresh' button but the Map still doesn't update after clicking 'refresh' (yes the progress spinner show).

Before I deleted the response on the front end, I'm viewing it on the response history 'map'. After I deleted the response on the front end, I clicked 'refresh' but the map didn't update (still showed the response). I waited another minute then refreshed again, no luck still. I switched to calendar view then returned to map view, only then the map response history updated.
